.github/workflows/test_openvino.yml

```diff
@@ -20,40 +18,59 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.12"]
+        test_pattern:
+          [
+            "*modeling.py",
+            "*modeling_diffusion.py",
+            "*modeling_sentence_transformers.py",
+            "*quantization*",
+            "*training*",
+            "*export*",
+          ]
```

Comment:
this one gives a maximum test time of 20 minutes, the alternative:
```
        test_pattern:
          [
            "test_modeling*",
            "test_quantization*",
            "test_training*",
            "test_export*",
          ]
```
takes 40 minutes, and the original `tests/openvino/*` takes 1 hour 10 minutes